images/Dockerfile.ubuntu

```diff
@@ -41,50 +42,51 @@ FROM ${FLAVOR}-${FLAVOR_RELEASE}-upstream AS common
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update \
-  && apt-get install -y --no-install-recommends \
+  && apt-get install -y --no-install-recommends --no-install-suggests \
     gnupg \
     software-properties-common \
   && add-apt-repository ppa:oibaf/test \
   && apt-get update \
-  && apt-get install -y --no-install-recommends \
+  && apt-get install -y --no-install-recommends --no-install-suggests \
     conntrack \
     console-data \
     cloud-guest-utils \
-    coreutils \
+    # coreutils \
     cryptsetup \
     curl \
     debianutils \
     dosfstools \
-    dracut \
-    dracut-network \
     e2fsprogs \
     fdisk \
     file \
     gawk \
     gdisk \
-    grub2-common \
     haveged \
     iproute2 \
     iptables \
     iputils-ping \
     jq \
-    kbd \
+    # added in https://github.com/kairos-io/kairos/commit/d413f0ec9c420a4847105e8b531e948cf4ce3176
+    # kbd \
     krb5-locales \
     less \
-    lldpd \
+    # can I remove this? it brings perl
+    # lldpd \
     lvm2 \
-    nano \
+    # there's pico
+    # nano \
     nbd-client \
-    neovim \
+    # vim is even bigger, so maybe just use pico
+    # neovim \ 
     nohang \
     open-iscsi \
     open-vm-tools \
     openssh-server \
     parted \
     rsync \
-    shim-signed \
-    snapd \
-    snmpd \
+    # snapd \
```

Comment:
Noice! I wonder if this will remove the /snap dir in the rootfs lol

Reply:
snapd is something actually very useful and I'd keep. 

Is taking much space?

A user could easily add something with snapd, and avoid messing up with apt

Reply:
@mudler it increases the size by 100Mb but I see why you'd like to keep it

Reply:
I can still boot the standard image with it so I'll keep it but it's space that the consumer will not have, we can decide later once Canvos can tell if the space is good enough